src/thunderbots/backend_output/grsim/grsim_backend.h

```diff
@@ -1,13 +1,62 @@
 #ifndef PROJECT_GRSIM_BACKEND_H
 #define PROJECT_GRSIM_BACKEND_H
+#include <boost/asio.hpp>
+#include <string>
 #include "backend_output/backend.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+
 
 class GrSimBackend : public Backend
 {
    public:
-    explicit GrSimBackend();
+    /**
+     * Creates a new grSim backend
+     *
+     * @param network_address The IP address to publish grSim commands to
+     * @param port The port to publish commands to
+     */
+    explicit GrSimBackend(std::string network_address, unsigned short port);
+    ~GrSimBackend();
+
+    /**
+     * Sends the given primitives to grSim
+     * @param primitives the list of primitives to send
+     */
     void sendPrimitives(const std::vector<Primitive> &primitives) override;
```

Comment:
If this is defined in `backend.h` we should remove the docstring here. Documentation should only every be defined in a single place.